src/installer/utils.py

```diff
@@ -67,3 +73,33 @@ def parse_wheel_filename(filename):
     if not wheel_info:
         raise ValueError("Not a valid wheel filename: {}".format(filename))
     return WheelFilename(*wheel_info.groups())
+
+
+def copyfileobj_with_hashing(
```

Comment:
Any bright ideas for a better name for this function? :3

Reply:
This looks to have been renamed, so marking as "resolved".